i18n/2017/categories/misc/es.md

```diff
@@ -0,0 +1 @@
+TODO: Need translation.
```

Comment:
If @michaelrambeau is k with the category message, i would love some help for the translation @jvalen ;)

Reply:
I'll be happy to help with the translation :wink:

Reply:
`Debido a que hay proyectos que no encajan a la perfección en las secciones anteriores, hemos creado este listado donde encontrarás los proyectos más populares del año, que a su vez, no pertenecen a ninguna de las otras categorías.`

Reply:
Thank you :smile: ! Can you also provide me the "Miscellaneous" translation for the category name ?